test/CheckedC/inferred-bounds/member-base.c

```diff
@@ -145,35 +145,6 @@ void f3(_Array_ptr<struct S> a : bounds(a, a + 7)) {
 // CHECK: `-ImplicitCastExpr {{0x[0-9a-f]+}} '_Array_ptr<struct S>' <LValueToRValue>
 // CHECK: `-DeclRefExpr {{0x[0-9a-f]+}} '_Array_ptr<struct S>' lvalue ParmVar {{0x[0-9a-f]+}} 'a' '_Array_ptr<struct S>'
 
-// CHECK: BinaryOperator {{0x[0-9a-f]+}} '_Ptr<int>' '='
```

Comment:
I should probably replace this test, as there's still an Implicit Cast going on in this code which needs testing, even though it is not tested in the version of the test I've removed these lines from.